rules/windows/image_load/image_load_office_excel_xll_load.yml

```diff
@@ -0,0 +1,22 @@
+title: Microsoft Excel Add-In Loaded
+id: c5f4b5cb-4c25-4249-ba91-aa03626e3185
+status: experimental
+description: Detects Microsoft Excel loading an Add-In (.xll) file
+references:
+    - https://www.mandiant.com/resources/blog/lnk-between-browsers
+author: Nasreddine Bencherchali (Nextron Systems)
+date: 2023/05/12
+tags:
+    - attack.execution
+    - attack.t1204.002
+logsource:
+    category: image_load
+    product: windows
+detection:
+    selection:
+        Image|endswith: '\excel.exe'
+        ImageLoaded|endswith: '.xll'
+    condition: selection
+falsepositives:
+    - The rules is only looking for ".xll" loads. So some false positives are expected with legitimate and allowed XLLs
```

Comment:
For low rules in general it doesn't make sense to utilise the falsepositive section. We expect a lot of non-malicious matches, otherwise it wouldn't be low. Maybe we add something like `N/A` for low rules?

Reply:
I'm starting to move similar rule to the TH folder. I think this one fits that definition.

We resolve it here and in my next PR I'll move it with a bunch of other rules to that folder and instead of `falsepositives` I'll add a new field that described what the hunt idea is looking for.